src/utils.js

```diff
@@ -94,8 +94,38 @@ function getActiveElement(document) {
   }
 }
 
+function getSelectionRange(element) {
+  if (isContentEditable(element)) {
+    const range = document.getSelection().getRangeAt(0)
+
+    return {selectionStart: range.startOffset, selectionEnd: range.endOffset}
```

Comment:
We should probably also make sure that the document's selection range is within the given element as well.